tests/fixtures/joint.py

```diff
@@ -80,3 +89,14 @@ async def joint_fixture(
                 rest_client=rest_client,
                 s3=s3_fixture,
             )
+
+
+def get_joint_fixture(scope: _ScopeName = "function"):
+    """Produce a joint fixture with desired scope"""
+    return pytest_asyncio.fixture(joint_fixture_function, scope=scope)
+
+
+joint_fixture = get_joint_fixture()
+mongodb_fixture = get_mongodb_fixture()
+kafka_fixture = get_kafka_fixture()
+s3_fixture = get_s3_fixture()
```

Comment:
This is not sensitive to order, i.e. joint_fixture does not need to be at the bottom?

Reply:
No, because it's just a function at this point. Pytest evaluates them during arrangement in the test module itself (I believe)